src/command_helpers.rs

```diff
@@ -18,13 +18,15 @@ use crate::{Error, ErrorKind, Object};
 pub(crate) struct CargoOutput {
     pub(crate) metadata: bool,
     pub(crate) warnings: bool,
+    pub(crate) debug: bool,
 }
 
 impl CargoOutput {
     pub(crate) const fn new() -> Self {
         Self {
             metadata: true,
             warnings: true,
+            debug: false,
```

Comment:
I don't really have a preference - would you rather this be enabled by default or use the env var instead?

Reply:
I'd like to use an env variable `CC_ENABLE_DEBUG_OUTPUT` - it would need a `cargo:rerun-if-env-changed=`

Reply:
Alright I got a little confused here - should there be no `debug` field at all then? And we just rely on this `CC_ENABLE_DEBUG_OUTPUT` env variable being changed with `cargo:rerun-if-env-changed`? I've just updated with what I think you asked for, but I might be totally wrong.

Reply:
I'd like to keep the debug field to allow override.

Also, the cargo:rerun-if-env should be in a CargoOutput::new, it should be only printed if metadata is true

Reply:
Ok, so what I've done is

print the `rerun-if-env-changed` in `CargoOutput::new`, but now it can no longer be const (which is fine, just a heads up)

set debug to if `CC_ENABLE_DEBUG_OUTPUT` is ok with `std::env::var(...).is_ok()`